packages/shared/lib/core/wallet/actions/getOutputRewards.ts

```diff
@@ -0,0 +1,7 @@
+import { getClient } from './getClient'
+
+export async function getOutputRewards(outputId: string): Promise<number> {
+    const client = await getClient()
+    const rewards = await client.getOutputManaRewards(outputId)
+    return Number(rewards)
```

Comment:
You are casting an object to a number :thinking: 
![image](https://github.com/iotaledger/firefly/assets/38158676/c11c9875-a0b5-4704-8b67-715f79a9fd62)

Reply:
I think you wanted to access the `rewards` property, right?